.github/workflows/zip.yml

```diff
@@ -249,20 +245,21 @@ jobs:
               cp quickstart-ios/crashlytics/LegacyCrashlyticsQuickstart/Firebase/upload-symbols quickstart-ios/crashlytics/LegacyCrashlyticsQuickstart
               chmod +x quickstart-ios/crashlytics/LegacyCrashlyticsQuickstart/run
               chmod +x quickstart-ios/crashlytics/LegacyCrashlyticsQuickstart/upload-symbols
-    # TODO(#8057): Restore Swift Quickstart
```

Comment:
Only a partial fix. Adding the LEGACY's gets the framework in the right directory, but there is still a mismatch somewhere between `Reachability` and ReachabilitySwift`. Going to continue to defer since the Firebase Swift frameworks are working.

Reply:
Yes. Good catch.